pkg/objectupdate/rte/rte.go

```diff
@@ -136,6 +136,13 @@ func DaemonSetArgs(ds *appsv1.DaemonSet, conf nropv1.NodeGroupConfig) error {
 		return fmt.Errorf("cannot modify the arguments for container %s", cnt.Name)
 	}
 
+	rtePauseEnabled := isRTEPauseEnabled(&conf)
+	klog.V(2).InfoS("DaemonSet update: RTEPause status", "daemonset", ds.Name, "enabled", rtePauseEnabled)
+	if rtePauseEnabled {
+		klog.V(4).InfoS("DaemonSet update: disable publishing NRTs updates")
```

Comment:
this should be rare enough that it deserves to be logged with `V(2)` level

Reply:
and let's put the log outside the if (for which we will need an `else` anyway, see below)